### PR TITLE
[refactor]: add permission token wrappers for private_blockchain

### PR DESCRIPTION
**Pull request by QuentinI**
### Description of the Change

- Added wrapper for tokens in private_blockchain
- Added `kagami tokens` subcommand to list those

### Benefits

Consistency mainly.

### Possible Drawbacks

None

### CODE REVIEW

File: tools/kagami/src/main.rs [hunk 1]
```diff
@@ -36,6 +36,7 @@ pub trait RunArgs<T: Write> {
 /// Tool generating the cryptographic key pairs, schema, genesis block and configuration reference.
 #[derive(StructOpt, Debug)]
 #[structopt(name = "kagami", version, author)]
+#[allow(variant_size_differences)]
```
Comment (appetrosyan) [inline tools/kagami/src/main.rs:39]: I wouldn't allow this. This needs to be fixed with boxing, and not hidden.

**Review by appetrosyan — CHANGES_REQUESTED** [review]

File: tools/kagami/src/main.rs (hunk 1 above)
Comment (QuentinI) [reply tools/kagami/src/main.rs:39]: I figured it doesn't make a lot of difference when it's instantiated only once, but I can box offending variants instead, sure.

**Review by appetrosyan — APPROVED** [review]

**Comment by appetrosyan** [comment]
Please adjust the PR title/commit message.

**Review by appetrosyan — APPROVED** [review]

File: tools/kagami/src/main.rs [hunk 2]
```diff
@@ -303,48 +303,89 @@ mod docs {
...
+        Private,
+        Public,
+    }
+
+    fn public_blockchain_tokens() -> Result<HashMap<String, HashMap<String, String>>> {
+        let mut schema = PredefinedPermissionToken::get_schema();
+
+        let enum_variants = match schema
+            .remove("iroha_permissions_validators::public_blockchain::PredefinedPermissionToken")
+            .ok_or_else(|| eyre!("Token enum not in schema"))?
```
Comment (outoftardis) [inline tools/kagami/src/main.rs:337]:
```suggestion
            .ok_or_else(|| eyre!("Token enum is not in schema"))?
```

File: tools/kagami/src/main.rs [hunk 3]
```diff
@@ -303,48 +303,89 @@ mod docs {
...
+            _ => bail!("Expected enum"),
+        };
+
+        enum_variants
+            .into_iter()
+            .map(|variant| {
+                let ty = variant.ty.ok_or_else(|| eyre!("Empty enum variant"))?;
+                let fields = match schema
+                    .remove(&ty)
+                    .ok_or_else(|| eyre!("Token not in schema"))?
```
Comment (outoftardis) [inline tools/kagami/src/main.rs:349]:
```suggestion
                    .ok_or_else(|| eyre!("Token is not in schema"))?
```

**Review by appetrosyan — APPROVED** [review]

**Review by ilchu — APPROVED** [review]
LGTM.

**Pull request merged**